nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/download/DownloadService.java

```diff
@@ -106,6 +121,28 @@ public long getLength() {
     String getDownloadUrl(String repositoryName, String docId, String xpath, String filename);
 
     /**
+     * Gets the URL to use to download the blobs identified by a storage key.
+     *
+     * <p>
+     * The URL is relative to the Nuxeo Web Application context.
+     * <p>
+     *
+     * @param blobs The key of stored blobs to download
+     * @return The download URL, something like {@code nxbigblobs/uuid}
```

Comment:
If you want to put an example then put it in the main part of the Javadoc, and make the example more precise.